config.example.json

```diff
@@ -5,7 +5,10 @@
     //ssl:The global ssl files setting
     "ssl": {
         "cert": "../../trantor/trantor/tests/server.pem",
-        "key": "../../trantor/trantor/tests/server.pem"
+        "key": "../../trantor/trantor/tests/server.pem",
+        "conf": [
+            ["min_protocol", "TLSv1.2"]
+        ]
```

Comment:
Please keep the conf option empty by default. you could comment off the example item.

Reply:
Agree. Also to be safe, please make sure an empty config `"conf" : []` would also work.

Reply:
Added, plz have a check.